releasenotes/notes/remove-tracer-deprecations-c0abaecda9c24b79.yaml

```diff
@@ -0,0 +1,12 @@
+---
+upgrade:
+  - |
+    Remove deprecated attribute ``ddtrace.Tracer.debug_logging``. Use ``logging.setLevel`` instead.
+  - |
+    Remove deprecated ``ddtrace.Tracer.__call__``.
+  - |
+    Remove deprecated method ``ddtrace.Tracer.global_excepthook``.
+  - |
+    Remove deprecated method ``ddtrace.Tracer.get_call_context``. Use ``ddtrace.Tracer.current_trace_context`` instead.
```

Comment:
Do we want a <code>:py:meth\`ddtrace.Tracer.current_trace_context\`</code> here? (if I got that correct)